Dockerfile

```diff
@@ -1,6 +1,4 @@
-# syntax=docker/dockerfile:1.2
-# Base is set up with the runtime dependencies and environment.
-FROM quay.io/centos/centos:stream8 AS base
+FROM registry.access.redhat.com/ubi8/ubi:latest AS base
```

Comment:
These changes to the Dockerfile should already be in the main branch. You might need to rebase in order to grab these changes.

Reply:
I dont understand why this is showing as a changed file in the PR when both files match.